templates/tfengine/components/cicd/main.tf

```diff
@@ -26,7 +26,7 @@
 {{- $hasScheduledJobs := false}}
 {{- $hasApplyJobs := false}}
 {{- range .envs}}
-  {{- if or (has .triggers.validate "run_on_schedule") (has .triggers.plan "run_on_schedule") (has .triggers.apply "run_on_schedule")}}
```

Comment:
why making this change? I think nested field should go the first argument of `has`